<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Chong Wu Lab</title>
</head>
<body>
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
    var pageTracker = _gat._getTracker("UA-88798670-1");
    pageTracker._trackPageview();
} catch(err) {}</script>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Chong Wu</div>
<div class="menu-item"><a href="index.html" class="current">Home</a></div>
<div class="menu-item"><a href="PI.html">PI</a></div>
<div class="menu-item"><a href="research.html">Research</a></div>
<div class="menu-item"><a href="grants.html">Grants</a></div>
<div class="menu-item"><a href="teaching.html">Teaching</a></div>
<div class="menu-item"><a href="students.html">Students</a></div>
<div class="menu-item"><a href="software.html">Software</a></div>
<div class="menu-item"><a href="talks.html">Presentations</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Chong Wu Lab</h1>
</div>
<h2>Intro</h2>
<p>Chong Wu is an Assistant Professor in the <a href="https://www.mdanderson.org/research/departments-labs-institutes/departments-divisions/biostatistics.html" target=&ldquo;blank&rdquo;>Department of Biostatistics</a> at <a href="https://www.mdanderson.org" target=&ldquo;blank&rdquo;>The University of Texas MD Anderson</a>. His research focuses on developing a new generation of data-driven methods and software to address challenges imposed by big and messy genomics data. Briefly, his research aims at 1) identifying putative causal biomarkers to gain insights into the genetic basis of complex diseases, particularly prostate cancer, pancreatic cancer, and Alzheimer's, and 2) enhancing risk predictions to advance precision medicine.</p>
<p><a href="https://github.com/ChongWuLab" target=&ldquo;blank&rdquo;>GitHub</a> <br /></p>
<p><a href="https://scholar.google.com/citations?hl=en&user=sWUsT2UAAAAJ" target=&ldquo;blank&rdquo;>Google Scholar</a></p>
<div class="infoblock">
<div class="blockcontent">
<p>We have two open Research Assistant positions and potentially one Postdoc position. The Research Assistant positions can be offered to students in MD Anderson, Rice, and UTHealth. Feel free to reach out (cwu18@mdanderson.org) if you are interested in joining our group.</p>
</div></div>
<h2>News</h2>
<ul>
<li><p>2022-11: <a href="https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-022-02583-y" target=&ldquo;blank&rdquo;>A polygenic risk score</a> has been published in BMC Medicine. Congratulations, Austin!</p>
</li>
<li><p>2022-10: <a href="https://www.nature.com/articles/s41467-022-34016-y" target=&ldquo;blank&rdquo;>The SUMMIT paper</a> has been published in Nature Communications. Congratulations, Zichen!</p>
</li>
<li><p>2022-09: Zhuo Meng received a travel award to attend Southern Regional Council on Statistics. Congratulations, Zhuo!</p>
</li>
<li><p>2022-08: Our alumna Dr. Shengjie Jiang will join The University of Texas at Dallas as an Assistant Professor of Instruction. Congratulations, Shengjie!</p>
</li>
<li><p>2022-08: We will join MD Anderson Biostatistics this September. We have enjoyed our time at Tallahassee and appreciate the support from the department! We will continue to collaborate with colleagues at FSU, and we look forward to our new journey at MD Anderson.</p>
</li>
<li><p>2022-07: Our <a href="https://reporter.nih.gov/search/fEzRBX-ZZk6Y7HE_Eo462A/project-details/10446594" target=&ldquo;blank&rdquo;>Prostate Cancer R01 (PI: Lang Wu and Chong Wu)</a> has been funded by NCI! This grant aims to identify likely causal proteins and improve the risk prediction for prostate cancer across Africans and Europeans. We are honored to lead this big project with Lang together.</p>
</li>
<li><p>2021-09: Our SUMMIT paper won a poster talk (top 10% among all posters) at <a href="https://www.ashg.org/meetings/2021meeting/" target=&ldquo;blank&rdquo;>ASHG 2021</a>. Congraulations, Zichen!</p>
</li>
<li><p>2021-09: Our <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/cac2.12205" target=&ldquo;blank&rdquo;>polygenic risk score in prostate cancer</a> paper has been published in <i>Cancer Communications</i>.</p>
</li>
<li><p>2021-05: Our <a href="https://www.nature.com/articles/s41436-021-01243-5" target=&ldquo;blank&rdquo;>COVID-19 integrative analysis</a> paper has been accepted by <i>Genetics in Medicine</i>.</p>
</li>
<li><p>2021-01: Our <a href="https://reporter.nih.gov/search/5WM4YWn1-UigU5OZ25HJ0w/project-details/10104707" target=&ldquo;blank&rdquo;>R03 proposal</a> (Novel Statistical Methods for Multi-omics Data Integration in Alzheimer's Disease) has been funded by NIA.</p>
</li>
</ul>
<h2>Selected Publications [<a href="research.html" target=&ldquo;blank&rdquo;>by topic</a>]</h2>
<p>* Corresponding author</p>
<ol>
<li><p>He, Y., Xu, G., <b>Wu, C.</b>, and Pan, W. (2021)<br /> <a href="https://arxiv.org/abs/1809.00411" target=&ldquo;blank&rdquo;>Asymptotically Independent U-Statistics in High-Dimensional Testing</a>.  <i>Annals of Statistics</i>, 49(1), 154-181.</p>
</li>
<li><p><b>Wu, C.</b>*, Bradley J., Li, Y., Wu, L., and Deng, HW. (2021)<br /> <a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btab045/6125360?redirectedFrom=fulltext" target=&ldquo;blank&rdquo;>A gene-level methylome-wide association analysis identifies novel Alzheimer’s disease genes</a>. <i>Bioinformatics</i>.</p>
</li>
<li><p><b>Wu, C.</b>*, Xu, G., Shen, X., and Pan, W.* (2020)<br /> <a href="https://jmlr.org/papers/v21/18-807.html" target=&ldquo;blank&rdquo;>A regularization-based adaptive test for high-dimensional generalized linear models</a>. <i>Journal of Machine Learning Research</i>, 21(128), 1-67.</p>
</li>
<li><p><b>Wu, C.</b> (2020) <br /> <a href="https://www.biorxiv.org/content/10.1101/758326v2.abstract" target=&ldquo;blank&rdquo;>Multi-trait genome-wide analyses of the brain imaging phenotypes in UK Biobank</a>. <i>Genetics</i>, 215(4), 947-958. <br />(Highlight in Genetics August issue; presented as a platform presentation at ASHG 2019 Annual Meeting)</p>
</li>
<li><p><b>Wu, C.</b>* and Pan, W.* (2019)<br /> <a href="https://doi.org/10.1093/bioinformatics/btz161" target=&ldquo;blank&rdquo;>Integration  of methylation QTL and enhancer-target gene maps with schizophrenia GWAS summary results identifies novel genes</a>.  <i>Bioinformatics</i>, 35(19), 3576-3583.</p>
</li>
<li><p><b>Wu, C.</b>, Chen, J., Kim, J., and Pan, W. (2016) <a href="https://link.springer.com/article/10.1186/s13073-016-0302-3" target=&ldquo;blank&rdquo;>An adaptive association test for microbiome data</a>. <i>Genome Medicine</i>, 8(1), 1-12.</p>
</li>
<li><p><b>Wu, C.</b>#, Kwon, S.#, Shen, X., and Pan, W. (2016). <a href="https://www.jmlr.org/papers/volume17/15-553/15-553.pdf" target=&ldquo;blank&rdquo;>A new algorithm and theory for penalized regression-based clustering</a>. <i>Journal of Machine Learning Research</i>, 17(1), 6479-6503. #, co-first author</p>
</li>
</ol>
<h2><a href="software.html" target=&ldquo;blank&rdquo;>Selected Software</a></h2>
<p>We put all recent developed software into a <a href="https://github.com/ChongWuLab" target=&ldquo;blank&rdquo;>lab github page</a>.</p>
<ul>
<li><p><a href="https://cran.r-project.org/web/packages/prclust/index.html" target=&ldquo;blank&rdquo;><i>prclust</i></a>: Penalized Regression-Based Clustering Methods;</p>
</li>
<li><p><a href="https://cran.r-project.org/web/packages/MiSPU/index.html" target=&ldquo;blank&rdquo;><i>MiSPU</i></a>: Microbiome Based Sum of Powered Score (MiSPU) Tests;</p>
</li>
<li><p><a href="https://github.com/ChongWu-Biostat/GLMaSPU" target=&ldquo;blank&rdquo;><i>GLMaSPU</i></a>: Adaptive Tests on High Dimensional Parameters in Generalized Linear Models;</p>
</li>
<li><p><a href="IWAS.html" target=&ldquo;blank&rdquo;>IWAS</a>: Imaging-Wide Association Study;</p>
</li>
<li><p><a href="TWAS.html" target=&ldquo;blank&rdquo;>TWAS</a>: Integrating eQTL and GWAS data;</p>
</li>
<li><p><a href="aSPUpath2.html" target=&ldquo;blank&rdquo;>aSPUpath2</a>: Integrating eQTL data with GWAS summary statistics in pathway-based analysis.</p>
</li>
</ul>
<h2>Grants</h2>
<p>Our research is generously supported by the following grants:</p>
<ul>
<li><p>R01 1R01CA263494-01A1, NCI <br />
PI, (Lang Wu and Chong Wu), 07-01-2022 to 06-30-2027<br />
<a href="https://reporter.nih.gov/search/_ngM5BoAiUKEDlNjCKqYAA/project-details/10446594" target=&ldquo;blank&rdquo;>Uncovering causal protein markers to improve prostate cancer etiology understanding and risk prediction in Africans and Europeans</a></p>
</li>
</ul>
<ul>
<li><p>R03 AG070669, NIA <br /> Contact PI, (Wu and Bradley), 01-01-21 to 12-31-22<br />
<a href="https://reporter.nih.gov/search/_ngM5BoAiUKEDlNjCKqYAA/project-details/10321679" target=&ldquo;blank&rdquo;>Novel Statistical Methods for Multi-omics Data Integration in Alzheimer's Disease</a></p>
</li>
</ul>
<ul>
<li><p>The Committee on Faculty Research Support, FSU<br /> PI, 05-07-20 to 06-30-20<br />
Novel Machine Learning Methods for Alzheimer’s Disease</p>
</li>
</ul>
<ul>
<li><p>First Year Assistant Professor Grant, FSU<br /> PI, 05-08-19 to 08-06-19<br />
Novel Statistical Methods for Transcriptome-wide Association Studies</p>
</li>
</ul>
<h2>Contact</h2>
<p>Chong Wu<br />
Assistant Professor<br />
Department of Biostatistics<br />
The University of Texas MD Anderson Cancer Center<br />
E-mail: cwu18@mdanderson.org</p>
<div id="footer">
<div id="footer-text">
Page generated 2022-08-23 00:27:40 CDT, by <a href="https://github.com/wsshin/jemdoc_mathjax" target="blank">jemdoc+MathJax</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
