<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Chong Wu</title>
</head>
<body>
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
    var pageTracker = _gat._getTracker("UA-88798670-1");
    pageTracker._trackPageview();
} catch(err) {}</script>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Chong Wu</div>
<div class="menu-item"><a href="index.html" class="current">Home</a></div>
<div class="menu-item"><a href="research.html">Research</a></div>
<div class="menu-item"><a href="teaching.html">Teaching</a></div>
<div class="menu-item"><a href="software.html">Software</a></div>
<div class="menu-item"><a href="talks.html">Presentations</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Chong Wu</h1>
</div>
<table class="imgtable"><tr><td>
<img src="chongwu.jpg" alt="Portrait of Chong Wu" width="230px" height="280px" />&nbsp;</td>
<td align="left"><p>Chong Wu <br /></p>
<p>Assistant Professor <br />
<a href="https://www.stat.fsu.edu" target=&ldquo;blank&rdquo;>Department of Statistics</a><br />
<a href="https://www.fsu.edu" target=&ldquo;blank&rdquo;>Florida State University</a><br /></p>
<p>Research interest: high-dimensional data, statistical genetics, integrative analysis, big data, and machine learning</p>
<p><a href="CV_ChongWu.pdf" target=&ldquo;blank&rdquo;>Curriculum Vitae (July, 2020)</a>
<br /></p>
<p><a href="https://github.com/ChongWu-Biostat" target=&ldquo;blank&rdquo;>GitHub</a> <br /></p>
<p><a href="https://scholar.google.com/citations?user=rF6BnvsAAAAJ&amp;hl=en" target=&ldquo;blank&rdquo;>Google Scholar</a></p>
</td></tr></table>
<h2>Short Bio</h2>
<p>Currently, I am an Assistant Professor in the <a href="https://www.stat.fsu.edu" target=&ldquo;blank&rdquo;>Department of Statistics</a> at <a href="https://www.fsu.edu" target=&ldquo;blank&rdquo;>Florida State University</a>. Prior to FSU, I was a biostatistics Ph.D. student at the <a href="http://twin-cities.umn.edu" target=&ldquo;blank&rdquo;>University of Minnesota</a>, co-advised by Profs. Weihua Guan and <a href="http://www.biostat.umn.edu/~weip/" target=&ldquo;blank&rdquo;>Wei Pan</a>. I received my bachelor's degree in Applied Math from <a href="http://english.hust.edu.cn" target=&ldquo;blank&rdquo;>Huazhong University of Science and Technology</a> in 2013.</p>
<p>My research interests are in statistical genetics and genomics, statistical learning, and machine learning. In the past a few years, I have obtained a diverse training in developing statistical methods and algorithms for analyzing multiple types of genetic and genomic data, including genome-wide association study (GWAS) data, DNA methylation data, and human microbiome data. My work includes gene- and pathway-based association testing and a new algorithm for clustering. To practice and facilitate reproducible research, I have developed and currently maintain several software and their online manuals (<a href="software.html" target=&ldquo;blank&rdquo;>see Software page</a>). My recent work involves integrative analysis of GWAS with multiple sources of omics data.</p>
<div class="infoblock">
<div class="blockcontent">
<p>I have extensive experience in analyzing genetics and genomics datasets (e.g., UK Biobank data, GTEx data, ARIC data, etc.). If you want to conduct GWAS or related research, you can contact me.</p>
<p>I am seeking highly self-motivated students to join my research group. If you are interested, feel free to email me or stop by my office. </p>
</div></div>
<h2>Recent Publications [<a href="research.html" target=&ldquo;blank&rdquo;>by topic</a>]</h2>
<p>* Corresponding author</p>
<ol>
<li><p>He, Y., Xu, G., <b>Wu, C.</b>, Pan, W. (2020)<br /> <a href="https://arxiv.org/abs/1809.00411" target=&ldquo;blank&rdquo;>Asymptotically Independent U-Statistics in High-Dimensional Testing</a>. Accepted by <i>Annals of Statistics</i>.</p>
</li>
<li><p><b>Wu, C.</b>* and Pan, W.* (2020)<br /> <a href="https://link.springer.com/article/10.1007/s00439-019-02098-2" target=&ldquo;blank&rdquo;>A powerful fine-mapping method for transcriptome-wide association studies</a>. <i>Human Genetics</i> 139, 199â€“213.</p>
</li>
<li><p><b>Wu, C.</b>*, Xu, G., Shen, X., and Pan, W.* (2020)<br /> A regularization-based adaptive test for high-dimensional generalized linear models. Accepted by <i>Journal of Machine Learning Research</i>.</p>
</li>
<li><p><b>Wu, C.</b> (2020) <br /> <a href="https://www.biorxiv.org/content/10.1101/758326v2.abstract" target=&ldquo;blank&rdquo;>Multi-trait genome-wide analyses of the brain imaging phenotypes in UK Biobank</a>. Accepted by <i>Genetics</i>. <br />(Highlight in Genetics August issue; presented as a platform presentation at ASHG 2019 Annual Meeting)</p>
</li>
<li><p><b>Wu, C.</b>* and Pan, W.* (2019)<br /> <a href="https://doi.org/10.1093/bioinformatics/btz161" target=&ldquo;blank&rdquo;>Integration  of methylation QTL and enhancer-target gene maps with schizophrenia GWAS summary results identifies novel genes</a>.  <i>Bioinformatics</i>, 35(19), 3576-3583.</p>
</li>
<li><p><b>Wu, C.</b>*, Xu, G., and Pan, W.* (2019).<br /> <a href="http://www3.stat.sinica.edu.tw/statistica/J29N4/J29N429/J29N429.html" target=&ldquo;blank&rdquo;>An adaptive test on high dimensional parameters in generalized linear models</a>. <i>Statistica Sinica</i>, 29, 2163-2186.</p>
</li>
<li><p><b>Wu, C.</b> and Pan, W. (2018). <br />
<a href="https://www.ncbi.nlm.nih.gov/pubmed/29411426" target=&ldquo;blank&rdquo;>Integrating eQTL data with GWAS summary statistics in pathwaybased analysis</a>. <i>Genetic Epidemiology</i>, 42.3: 303-316.<br />
(Presented as a poster talk at ASHG 2017 Annual Meeting.)</p>
</li>
</ol>
<h2><a href="software.html" target=&ldquo;blank&rdquo;>Selected Software</a></h2>
<ul>
<li><p><a href="https://cran.r-project.org/web/packages/prclust/index.html" target=&ldquo;blank&rdquo;><i>prclust</i></a>: Penalized Regression-Based Clustering Methods;</p>
</li>
<li><p><a href="https://cran.r-project.org/web/packages/MiSPU/index.html" target=&ldquo;blank&rdquo;><i>MiSPU</i></a>: Microbiome Based Sum of Powered Score (MiSPU) Tests;</p>
</li>
<li><p><a href="https://github.com/ChongWu-Biostat/GLMaSPU" target=&ldquo;blank&rdquo;><i>GLMaSPU</i></a>: Adaptive Tests on High Dimensional Parameters in Generalized Linear Models;</p>
</li>
<li><p><a href="IWAS.html" target=&ldquo;blank&rdquo;>IWAS</a>: Imaging-Wide Association Study;</p>
</li>
<li><p><a href="TWAS.html" target=&ldquo;blank&rdquo;>TWAS</a>: Integrating eQTL and GWAS data;</p>
</li>
<li><p><a href="aSPUpath2.html" target=&ldquo;blank&rdquo;>aSPUpath2</a>: Integrating eQTL data with GWAS summary statistics in pathway-based analysis.</p>
</li>
</ul>
<h2>Students</h2>
<p>Currently, I am working with a few PhD students.</p>
<h2>Teaching</h2>
<p>For me, teaching can be described as a rewarding and fulfilling experience. I am the instructor for the following courses.</p>
<ul>
<li><p>2018 Fall to 2020 Spring: STAT 4321/4323 Introduction of Math Stat</p>
</li>
<li><p><a href="exampleforhighd.R" target=&ldquo;blank&rdquo;>Codes for Statistical Learning and Data Mining</a></p>
</li>
</ul>
<h2><a href="talks.html" target=&ldquo;blank&rdquo;>Selected Presentations</a></h2>
<p>I enjoy sharing ideas and presenting my works. I attend JSM, ENAR, and ASHG regularly.</p>
<ul>
<li><p><a href="aMAT.pdf" target=&ldquo;blank&rdquo;>Multi-trait genome-wide analyses of the brain imaging phenotypes in UK Biobank</a></p>
<ul>
<li><p>Platform presentation at ASHG 2019</p>
</li></ul>
</li>
<li><p><a href="ChongWu_IGES2018.pdf" target=&ldquo;blank&rdquo;>Integration of methylation QTL and enhancer-target gene maps with schizophrenia GWAS summary results identifies novel genes</a></p>
<ul>
<li><p>Oral presentation at IGES 2018</p>
</li></ul>
</li>
<li><p><a href="Chong_Wu_JSM.pdf" target=&ldquo;blank&rdquo;>An adaptive association test for microbiome data</a></p>
<ul>
<li><p>Presented at 2016 JSM, ENAR</p>
</li></ul>
</li>
<li><p><a href="presentation_aSPU_theory.pdf" target=&ldquo;blank&rdquo;>An adaptive test on high dimensional parameters in GLMs</a></p>
<ul>
<li><p>Presented at 2017 JSM, ENAR</p>
</li>
</ul>

</li>
</ul>
<h2>Awards</h2>
<ul>
<li><p><b>ENAR Distinguished Student Paper Award</b>, 2019</p>
</li>
<li><p>James R. Boen Student Achievement Award, Division of Biostatistics, University of Minnesota, 2018</p>
</li>
<li><p>Pre-Doctoral Trainee Award, Association of Chinese Geneticists in America (ACGA), 2017</p>
</li>
<li><p>Elected to Delta Omega (Public Health Honorary Society), 2017</p>
</li>
<li><p>Elected to Simga Xi (The Scientific Research Society), 2017</p>
</li>
<li><p><b>JSM Distinguished Student Paper Award</b> on Statistics in Genomics and Genetics Section, 2016</p>
</li>
<li><p><b>Doctoral Dissertation Fellowship</b>, University of Minnesota, 2016</p>
</li>
<li><p>Travel Award, Computational Neuroscience Summer School, Statistical and Applied Mathematical Sciences Institute, 2015</p>
</li>
<li><p>Dean's Ph.D. Scholarship, University of Minnesota, 2013</p>
</li>
<li><p>Honorable Mention in Mathematical Contest in Modeling, Consortium for Mathematics and Its Application, 2012</p>
</li>
<li><p>National Scholarship, Ministry of Education, China, 2011</p>
</li>
</ul>
<h2>Contact</h2>
<p>Chong Wu<br />
201C OSB, 117 N. Woodward Ave.<br />
P.O. Box 3064330, Tallahassee, FL, 32306-4330<br />
E-mail: cwu3@fsu.edu</p>
<div id="footer">
<div id="footer-text">
Page generated 2020-07-15 12:43:34 CDT, by <a href="https://github.com/wsshin/jemdoc_mathjax" target="blank">jemdoc+MathJax</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
