<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title></title>
</head>
<body>
<div id="layout-content">
<h1>IWAS</h1>
<h2>Imaging-Wide Association Study</h2>
<p>IWAS is a new tool for performing an image-wide association study by incorporating imaging endophenotype based external weights into GWAS using only summary statistics. The goal is to identify new genes that are associated with a trait of interest. We provide precomputed external weights (based on gray matter volume as the endophenotype) from ADNI study to facilitate this analysis. Please cite the following manuscript for using IWAS method:</p>
<div class="infoblock">
<div class="blockcontent">
<p>Xu et al. &ldquo;Imaging-wide association study: integrating imaging endophenotypes in GWAS&rdquo; 2017 Neuroimage, under revision.</p>
</div></div>
<p>For questions or comments regarding methods, contact Wei Pan (<a href="mailto:weip@biostat.umn.edu" target=&ldquo;blank&rdquo;>weip@biostat.umn.edu</a>);
For questions or comments regarding codes, contact Chong Wu (<a href="mailto:chongwu@umn.edu" target=&ldquo;blank&rdquo;>chongwu@umn.edu</a>).</p>
<h2>Installation</h2>
<ul>
<li><p>Download and unpack the <a href="Scripts," target=&ldquo;blank&rdquo;>reference data, and external weights</a> (Or download directly using clone or download option at GitHub.).</p>
</li>
</ul>
<div class="codeblock">
<div class="blockcontent"><pre>
wget https://github.com/gusevlab/fusion_twas/archive/master.zip
unzip master.zip
cd fusion_twas-master
</pre></div></div>
<ul>
<li><p>install <a href="https://www.cog-genomics.org/plink2" target=&ldquo;blank&rdquo;>plink</a> and put it into the working directory.</p>
</li>
<li><p>Launch R and install required libraries:</p>
</li>
</ul>
<div class="codeblock">
<div class="blockcontent"><pre>
install.packages('optparse','RColorBrewer','data.table','matlib','Rcpp','RcppArmadillo','bigmemory','mvtnorm','MASS')
install.packages('plink2R-master/plink2R/',repos=NULL)
if (!require("devtools"))
  install.packages("devtools")
devtools::install_github("ChongWu-Biostat/aSPU2")
devtools::install_github("gabraham/plink2R/plink2R")
</pre></div></div>
<h2>Typical analysis and output</h2>
<p>The IWAS analysis takes imaging endophenotype based external weights and disease GWAS summary statistics to identify significant genes.</p>
<p>We will use IGAP Alzheimer's summary as an example to illustrate how to use our methods. This example assumes you have setup the required environment as illustrated in the previous section. All analysis are based on R/3.3.1.</p>
<h3>Input: GWAS summary statistics</h3>
<p>At a minimum, we need a summary rds file with a header row containing the following fields:</p>
<ol>
<li><p>SNP_map – SNP identifier (CHR:BP)</p>
</li>
<li><p>A1 – first allele (effect allele, should be capitalized)</p>
</li>
<li><p>A2 – second allele (other allele, should be capitalized)</p>
</li>
<li><p>Z – Z-scores, sign with respect to A1.</p>
</li>
</ol>
<p>Additional columns are allowed but will be ignored. We recommend removing the additional columns before analysis to save space.</p>
<p><b>Note:</b> The performance of IWAS depends on the density of summary-level data. We highly recommend running IWAS with raw summary-level data. Pre-process step such as pruning and restricting to top SNPs may harm the performance.</p>
<h3>Input: external weights</h3>
<p>We pre-computed the external weights based on ADNI-1 or ADNI-2 data. The external weights data is stored in the .<i>WEIGHTS</i> which provides the external weights for each gene, and their corresponding physical positions.</p>
<h3>Performing the IWAS</h3>
<p>After we prepared the data, we can run IWAS via the following single line.</p>
<div class="codeblock">
<div class="blockcontent"><pre>
Rscript IWAS.R \
--sumstats ./Example/IGAP_chr22.rds \
--out ./Example/example_res.rds \
--weights ./WEIGHTS/ADNI1_wgt.rds \
--ref_ld ./LDREF/hapmap_CEU_r23a_hg19 \
--gene.list ./Example/gene_list.tx \
--test_type aSPU \
--weight_type ST31CV
</pre></div></div>
<p>This should take around one or two minutes, and you will see some intermediate steps on the screen. If everything worked, you would see a file <i>example_res.rds</i> under the <i>.Example</i> directory.</p>
<p>Under the hood, we perform the following steps: (1) unify the GWAS and reference SNPs and remove ambiguous SNPs. (2) impute GWAS Z-scores for any missing reference SNPs via the IMPG algorithm; (3) perform IWAS; (4) report results and store it in the working directory.</p>
<h3>Output: Gene-disease association</h3>
<p>The results are stored in the user-defined output file. For illustration, we explain the meaning of each entry in the first two lines.</p>
<table id="VariExp">
<tr class="r1"><td class="c1">number </td><td class="c2"> Column   </td><td class="c3"> Value  </td><td class="c4"> Explanations </td></tr>
<tr class="r2"><td class="c1">1 </td><td class="c2"> gene         </td><td class="c3"> AK</td><td class="c4"> Feature/gene identifier, taken from gene_list file </td></tr>
<tr class="r3"><td class="c1">2 </td><td class="c2"> CHR        </td><td class="c3"> 22     </td><td class="c4"> Chromosome </td></tr>
<tr class="r4"><td class="c1">3 </td><td class="c2"> P0            </td><td class="c3"> 11</td><td class="c4"> Gene start (from hg19 list from plink website) </td></tr>
<tr class="r5"><td class="c1">4 </td><td class="c2"> P1        </td><td class="c3">   12  </td><td class="c4"> Gene end (from hg19 list from plink website) </td></tr>
<tr class="r6"><td class="c1">5 </td><td class="c2"> #nonzero_SNPs </td><td class="c3">11 </td><td class="c4"> Number of non-zero weight SNPs </td></tr>
<tr class="r7"><td class="c1">6 </td><td class="c2"> TWAS_asy </td><td class="c3"> </td><td class="c4"> TWAS p-value with imaging based external weight. The p-value is based on asymptotic distribution. </td></tr>
<tr class="r8"><td class="c1">7 </td><td class="c2"> SSU_asy </td><td class="c3"> </td><td class="c4"> SSU p-value with imaging based external weight. The p-value is based on asymptotic distribution. </td></tr>
<tr class="r9"><td class="c1">8-23</td><td class="c2"> SPU </td><td class="c3"> </td><td class="c4"> SPU or aSPU p-values. The results are based on simulations. 
</td></tr></table>
<h2>External Weights</h2>
<p>We put the external weights into GitHub. You can compute your own weights as well. To facilitate this step, we put our source file into the folder <i>.Weight/conWeights.R</i>. Our method is based on elastic net, and you can use other methods to compute the weights. Currently, we provide the pre-computed weights for the following 14 imaging endophenotype. You can define which image endophenotype you want to use by weight_type options. </p>
<table id="TABLENAME">
<tr class="r1"><td class="c1">left   </td><td class="c2"> right  </td><td class="c3"> ROI name </td></tr>
<tr class="r2"><td class="c1">ST31CV </td><td class="c2"> ST90CV </td><td class="c3"> Inferior parietal </td></tr>
<tr class="r3"><td class="c1">ST32CV </td><td class="c2"> ST91CV </td><td class="c3"> Inferior temporal </td></tr>
<tr class="r4"><td class="c1">ST39CV </td><td class="c2"> ST98CV </td><td class="c3"> Medial orbitofrontal </td></tr>
<tr class="r5"><td class="c1">ST44CV </td><td class="c2"> ST103CV </td><td class="c3"> Parahippocampal </td></tr>
<tr class="r6"><td class="c1">ST52CV </td><td class="c2"> ST111CV </td><td class="c3"> Precuneus </td></tr>
<tr class="r7"><td class="c1">T50CV </td><td class="c2"> ST109CV </td><td class="c3"> Posterior cingulate </td></tr>
<tr class="r8"><td class="c1">ST29SV </td><td class="c2"> ST88SV  </td><td class="c3"> Hippocampus 
</td></tr></table>
<h2>Command-line parameters</h2>
<p>write something here.</p>
<h2>FAQ</h2>
<p><b>IWAS looks similar to TWAS and PrediXcan. What's the difference between them?</b></p>
<p>Unlike TWAS and PrediXcan uses gene expression as the endophenotype, IWAS uses imaging endophenotype, which might be more powerful when analyzing some psychiatry related traits, such as AD. By noting that TWAS and PrediXcan are the same as weighted Sum test with external gene expression based weights, we propose to use aSPU, a more powerful and adaptive test, to conduct the association test. Since aSPU covers Weighted Sum test as a special scenario, we can get the results for TWAS or PrediXcan after running our methods as well. We demonstrate that our new method is more powerful and identify some new genes in some real data analyses. We expect this is generally true and hope you can let us know your results if you apply both TWAS and IWAS on your real data analyses.</p>
<p><b>Can I use other reference data?</b></p>
<p>Yes. Other references such as 1000 Genomes, can be used as well. We will upload the 1000 Genomes with only HapMap SNPs set in the near future. Please check this website later for updating.</p>
<p><b>What related software is available?</b></p>
<p>Two methods are highly correlated with two well-known groups.</p>
<p>The MetaXcan and PrediXcan developed by Im lab perform gene-based association tests with and without summary data. </p>
<p>TWAS by Gusev performs gene-based association tests only with summary data. </p>
<p>In our mind, MetaXcan and TWAS are very similar. The main difference between TWAS and MetaXcan is that they use a slightly different way and different reference data to construct weight. We recommend you check their website to download other weights as well.</p>
<p>We submit a paper to Genetics with extension to TWAS or PrediXcan. The main idea is that, instead of applying weighted Sum test, we apply aSPU. We demonstrate that aSPU is more powerful than TWAS and PrediXcan. </p>
<h2>Acknowledgements</h2>
<p>We are grateful to the external reference data, and all of the hard work involved in the study. Without the efforts of these groups and sharing the individual level data, we are unable to construct the external weights, and our methods will be useless. </p>
<div class="infoblock">
<div class="blockcontent">
<p>Data collection and sharing for this project was funded by the Alzheimer’s Disease Neuroimaging Initiative (ADNI) (National Institutes of Health Grant U01 AG024904) and DOD
ADNI (Department of Defense award number W81XWH-12-2-0012). ADNI is funded by the National Institute on Aging, the National Institute of Biomedical Imaging and Bioengineering, and through generous contributions from the following: Alzheimer’s Association; Alzheimers Drug Discovery Foundation; Araclon Biotech; BioClinica, Inc.; Biogen Idec Inc.; Bristol-Myers Squibb Company; Eisai Inc.; Elan Pharmaceuticals, Inc.; Eli Lilly and Company; EuroImmun; F. Ho↵mann-La Roche Ltd and its a liated company Genentech, Inc.; Fujirebio; GE Healthcare; IXICO Ltd.; Janssen Alzheimer Immunotherapy Research &amp; Development, LLC.; Johnson &amp; Johnson Pharmaceutical Research &amp; Development LLC.; Medpace, Inc.; Merck &amp; Co., Inc.; Meso Scale Diagnostics, LLC.; NeuroRx Research; Neurotrack Technologies; Novartis Pharmaceuticals Corporation; Pfizer Inc.; Piramal Imaging; Servier; Synarc Inc.; and Takeda Pharmaceutical Company. The Canadian Institutes of Rev December 5, 2013 Health Research is providing funds to support ADNI clinical sites in Canada. Private sector contributions are facilitated by the Foundation for the National Institutes of Health (www.fnih.org). The grantee organization is the Northern California Institute for Research and Education, and the study is coordinated by the Alzheimer’s Disease Cooperative Study at the University of California, San Diego. ADNI data are disseminated by the Laboratory for Neuro Imaging at the University of Southern California.</p>
</div></div>
</div>
</body>
</html>
