<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Research</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Chong Wu</div>
<div class="menu-item"><a href="index.html">Home</a></div>
<div class="menu-item"><a href="research.html" class="current">Research</a></div>
<div class="menu-item"><a href="talks.html">Talks</a></div>
<div class="menu-item"><a href="software.html">Software</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Research</h1>
</div>
<h2>Supervised learning</h2>
<ul>
<li><p><b>Wu C.</b> &amp; Pan W.<br />
Network-based support vector machine with min-based penalties.<br />
In preparation</p>
</li>
<li><p><b>Wu, C.</b>*,Kwon, S.*,Shen X., &amp; Pan W. (2016) (*: equal contribution)<br />
<a href="http://jmlr.org/papers/volume17/15-553/15-553.pdf" target=&ldquo;blank&rdquo;>A New Algorithm and Theory for Penalized Regression-based Clustering.</a><br />
<a href="http://jmlr.org" target=&ldquo;blank&rdquo;>Journal of Machine Learning Research</a>, 17, 1-25</p>
</li>
</ul>
<h2>High dimensional testing</h2>
<p>Due to the surge of high dimensional data in many fields, ranging from business to genetics, high dimensional testing problems have been increasingly studied. In high dimensional situations with the dimension of data \(p\) larger than the sample size \(n\), traditional multivariate tests, such as the likelihood ratio test and the Wald test, break down. More importantly, since the sample size \(n\) is often relatively small and the underlying truth is unknown, existing methods may not powerful under certain scenarios.</p>
<p>An intriguing question is: How can we propose an adaptive and powerful that maintains power across various scenarios? My research tries to answer this question from both theory and application perspectives.</p>
<h3>Theory papers</h3>
<ul>
<li><p><b>Wu, C.</b>, Xu, G., &amp; Pan W.<br />
An Adaptive Test on High Dimensional Parameters in Generalized Linear Models.<br />
Journal of Machine Learning Research (JMLR), submitted</p>
</li>
<li><p><b>Wu C.</b>, Xu G., &amp; Pan W.<br />
An adaptive and powerful test for high dimensional covariance matrices.<br />
In preparation</p>
</li>
</ul>
<h3>Applied papers</h3>
<ul>
<li><p><b>Wu, C.</b>, Chen, J., Kim, J., &amp; Pan, W. (2016).<br />
<a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-016-0302-3" target=&ldquo;blank&rdquo;>An adaptive association test for microbiome data.</a><br />
<a href="https://genomemedicine.biomedcentral.com" target=&ldquo;blank&rdquo;>Genome Medicine</a>, 8(1), 1.<br />
(<b>Distinguished Student Paper Award in Genomics and Genetics Section</b>, ASA Sections on Genomics and Genetics, 2016)</p>
</li>
</ul>
<h2>Statistical genetics</h2>
<h3>Methylation data</h3>
<p>DNA methylation is a widely studied epigenetic mechanism. Incorporating epigenetic variation into genetic studies of these diseases represents a powerful and effective way to account for environmental influences that gene-sequence-based studies of disease do not. Due to the known and unknown confounders, many false positives have been found.</p>
<p>An intriguing question is: How can we propose novel statistical methods to facilitate the analysis of high throughput DNA methylation data? My research tries to answer this question from improving the data quality perspective.</p>
<ul>
<li><p><b>Wu C.</b>, Pankow J. S., Demerath E. W.,&hellip;, &amp; Guan W.<br />
Iterative PCA in epigenome-wide association studies.<br />
In preparation</p>
</li>
<li><p><b>Wu, C.</b>, Demerath, E. W., Pankow, J. S., Bressler, J., Fornage, M., Grove, M. L., &hellip; &amp; Guan, W. (2016).<br />
<a href="http://www.tandfonline.com/doi/full/10.1080/15592294.2016.1145328" target=&ldquo;blank&rdquo;>Imputation of missing covariate values in epigenome-wide analysis of DNA methylation data.</a><br />
<a href="http://www.tandfonline.com/toc/kepi20/current" target=&ldquo;blank&rdquo;>Epigenetics</a>, 11(2), 132-139.</p>
</li>
<li><p>Bose, M., <b>Wu, C.</b>, Pankow, J. S., Demerath, E. W., Bressler, J., Fornage, M., &hellip; &amp; Kao, W. H. (2014).<br />
<a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-312" target=&ldquo;blank&rdquo;>Evaluation of microarray-based DNA methylation measurement using technical replicates: the Atherosclerosis Risk In Communities (ARIC) Study.</a><br />
<a href="https://bmcbioinformatics.biomedcentral.com" target=&ldquo;blank&rdquo;>BMC Bioinformatics</a>, 15(1), 312.</p>
</li>
</ul>
<h3>Microbiome data</h3>
<p>The genomes of human microbes, and the way they interact with the human host, are collectively termed the microbiome. Human microbiome association studies aim to de- tect an association of human microbiome diversity with a phenotype of interest, such as disease status; this can improve our understanding of complex traits and diseases. With my co-advisor Prof. Wei Pan, I proposed a powerful data-driven approach to conducting human microbiome association studies.</p>
<ul>
<li><p><b>Wu, C.</b>, Chen, J., Kim, J., &amp; Pan, W. (2016).<br />
<a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-016-0302-3" target=&ldquo;blank&rdquo;>An adaptive association test for microbiome data.</a><br />
<a href="https://genomemedicine.biomedcentral.com" target=&ldquo;blank&rdquo;>Genome Medicine</a>, 8(1), 1.<br />
(<b>Distinguished Student Paper Award in Genomics and Genetics Section</b>, ASA Sections on Genomics and Genetics, 2016)</p>
</li>
</ul>
</td>
</tr>
</table>
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
