<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Research</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Chong Wu</div>
<div class="menu-item"><a href="index.html">Home</a></div>
<div class="menu-item"><a href="research.html" class="current">Research</a></div>
<div class="menu-item"><a href="talks.html">Presentations</a></div>
<div class="menu-item"><a href="software.html">Software</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Research</h1>
</div>
<h2>Machine learning</h2>
<ol>
<li><p><b>Wu, C.</b>*,Kwon, S.*,Shen X., &amp; Pan W. (2016) (*: equal contribution)<br />
<a href="http://jmlr.org/papers/volume17/15-553/15-553.pdf" target=&ldquo;blank&rdquo;>A New Algorithm and Theory for Penalized Regression-based Clustering.</a> <i><a href="http://jmlr.org" target=&ldquo;blank&rdquo;>Journal of Machine Learning Research</a></i>, 17, 1-25</p>
</li>
<li><p>Liu, B., <b>Wu, C.</b>, Shen, X., and Pan, W. (2017+). <br /> <a href="https://www.biorxiv.org/content/early/2017/03/17/117473" target=&ldquo;blank&rdquo;>A novel and efficient algorithm for de novo discovery of mutated driver pathways</a>. Accepted by <i>Annals of Applied Statistics</i>, early online.</p>
</li>
<li><p><b>Wu C.</b> &amp; Pan W.<br />
Network-based support vector machine with a new penalty. In preparation.</p>
</li>
</ol>
<h2>Methods and applications in genetics and genomics</h2>
<h3>Integrative analysis of GWAS and other omic data.</h3>
<p>Recently, new gene-based GWAS tests were proposed to impute gene expression and then test its association with a trait in a GWAS dataset. We propose some more powerful tests along this direction.</p>
<ol>
<li><p>Xu, Z., <b>Wu, C.</b>, Wei, P., and Pan, W. (2017+).<br /> <a href="http://www.genetics.org/content/genetics/early/2017/09/11/genetics.117.300270.full.pdf" target=&ldquo;blank&rdquo;>A powerful framework for integrating eQTL and GWAS summary data</a>. Accepted by <i>Genetics</i>, early online. (IF: 4.6)</p>
</li>
<li><p>Xu, Z., <b>Wu, C.</b>, Pan, W., and Alzheimer’s Disease Neuroimaging Initiative (ADNI). (2017). <br /> <a href="https://www.ncbi.nlm.nih.gov/pubmed/28736311" target=&ldquo;blank&rdquo;>Imaging-wide association study: integrating imaging endophenotypes in GWAS</a>. <i>NeuroImage</i>, 159:159–169. <br /> (IF: 5.8.  is paper won a platform presentation at the American Society of Human Genetics (ASHG) 2017 Annual Meeting.)</p>
</li>
<li><p><b>Wu, C.</b> and Pan, W. (2017+). <br /> Integrating eQTL data with GWAS summary statistics in pathway-based analysis. Submitted. <br />(This paper won a poster talk (top 24 posters among about 3000 posters) at ASHG 2017 Annual Meeting.)</p>
</li>
</ol>
<p>Related software:</p>
<ol>
<li><p><a href="IWAS.html" target=&ldquo;blank&rdquo;>IWAS</a>: Imaging-Wide Association Study;</p>
</li>
<li><p><a href="TWAS.html" target=&ldquo;blank&rdquo;>TWAS</a>: Integrating eQTL and GWAS data;</p>
</li>
<li><p><a href="aSPUpath2.html" target=&ldquo;blank&rdquo;>aSPUpath2</a>: Integrating eQTL data with GWAS summary statistics in pathway-based analysis.</p>
</li>
</ol>
<h3>DNA methylation data analysis.</h3>
<p>DNA methylation is a widely studied epigenetic mechanism. The Athero-sclerosis Risk in Communities (ARIC) study measures DNA methylation over 480,000 methylation markers obtained from about 3,000 subjects. I develop new methods to solve challenges arising from the ARIC DNA methylation data.</p>
<ol>
<li><p><b>Wu, C.</b>, Demerath, E. W., Pankow, J. S., Bressler, J., Fornage, M., Grove, M. L., Chen, W., and Guan, W. (2016). <br />  I<a href="http://www.tandfonline.com/doi/full/10.1080/15592294.2016.1145328" target=&ldquo;blank&rdquo;>Imputation of missing covariate values in epigenome-wide analysis of DNA methylation data</a>. <i>Epigenetics</i>, 11(2):132–139. <br /> (IF: 4.4, the official journal of the DNA Methylation Society.)</p>
</li>
<li><p>Bose, M., <b>Wu, C.</b>, Pankow, J. S., Demerath, E. W., Bressler, J., Fornage, M., Grove, M. L., Mosley, T. H., Hicks, C., North, K., Kao, W. H., Zhang, Y., Boerwinkle, E., and Guan, W. (2014).<br />  <a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-312" target=&ldquo;blank&rdquo;>Evaluation of microarray-based DNA methylation measurement using technical replicates: the Atherosclerosis Risk In Communities (ARIC) Study</a>. <i>BMC Bioinformatics</i>, 15(1):1–10. (IF: 2.4)</p>
</li>
</ol>
<h3>Human microbiome data analysis.</h3>
<ol>
<li><p><b>Wu, C.</b>, Chen, J., Kim, J., and Pan, W. (2016).<br />  <a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-016-0302-3" target=&ldquo;blank&rdquo;>An adaptive association test for microbiome data</a>. <i>Genome Medicine</i>, 8(1):1–12.<br />  (IF: 7.1.  is paper won the 2016 Joint Statistical Meetings (JSM) Distinguished Student Paper Award on Statistics in Genomics and Genetics Section.)</p>
</li>
</ol>
<h3>Others</h3>
<ol>
<li><p>Park, J. Y., <b>Wu, C.</b>, and Pan, W. (2017+).<br /> An adaptive gene-level association test for pedigree data.
Accepted by <i>BMC Proceedings</i>, a special issue for the Genetic Analysis Workshop (GAW 20).</p>
</li>
<li><p><b>Wu, C.</b>, Park, J.Y., Guan, W., and Pan, W. (2017+). <br /> A powerful gene-based test for methylation
data. Accepted by BMC Proceedings, a special issue for the Genetic Analysis Workshop (GAW 20).</p>
</li>
<li><p>Park, J.Y., <b>Wu, C.</b>, Basu, S., McGue, M., and Pan, W. (2017+). Adaptive SNP set association testing in generalized linear mixed models with application to family studies. <i>Behavior Genetics</i>, revised and resubmitted.</p>
</li>
</ol>
<h2>Hypothesis tests on high-dimensional parameters in generalized linear models (GLMs).</h2>
<p>Regarding hypothesis testing, I have developed a new adaptive test for testing high-dimensional parameters under GLMs with low-dimensional nuisance parameters or with high-dimensional nuisance parameters.</p>
<ol>
<li><p><b>Wu, C.</b>, Xu, G., Shen, X., and Pan, W. (2017+).<br />
An adaptive test on a high-dimensional parameter in the presence of a high-dimensional nuisance parameter in GLM with application to detect gene-environment interactions. Manuscript.<br /> (Job talk manuscript, to be submitted to <i>Journal of the American Statistical Association</i>.)</p>
</li>
<li><p><b>Wu, C.</b>*, Xu, G., and Pan, W.* (2017+). <br /> An adaptive test on high dimensional parameters in
generalized linear models. <i>Statistica Sinica</i>, revised and resubmitted. (* Corresponding author)</p>
</li>
</ol>
<h2>Collaborative research</h2>
<ol>
<li><p>Zhu, L., Li, Y., Chen, Y., Carrera, C., <b>Wu, C.</b>, and Fork, A. (2017+).<br /> Comparison between two
post-dentin bond strength measurement methods. <i>Scientific Reports</i>, revised and resubmitted.</p>
</li>
</ol>
</td>
</tr>
</table>
</body>
</html>
