<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title></title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Chong Wu</div>
<div class="menu-item"><a href="index.html">Home</a></div>
<div class="menu-item"><a href="PI.html">PI</a></div>
<div class="menu-item"><a href="research.html">Research</a></div>
<div class="menu-item"><a href="grants.html">Grants</a></div>
<div class="menu-item"><a href="teaching.html">Teaching</a></div>
<div class="menu-item"><a href="students.html">Students</a></div>
<div class="menu-item"><a href="software.html" class="current">Software</a></div>
<div class="menu-item"><a href="talks.html">Presentations</a></div>
</td>
<td id="layout-content">
<h2>Softwares</h2>
<p>I have developed and currently maintain the following software. You can get the latest version from my <a href="https://github.com/ChongWu-Biostat" target=&ldquo;blank&rdquo;>GitHub repository</a>.</p>
<ul>
<li><p><a href="https://cran.r-project.org/web/packages/prclust/index.html" target=&ldquo;blank&rdquo;>prclust</a>: R package that provides two algorithms for fitting the penalized regression-based clustering (PRclust). The corresponding paper is <a href="http://jmlr.org/papers/volume17/15-553/15-553.pdf" target=&ldquo;blank&rdquo;>Wu, Kwon, Shen and Pan, 2016</a>.</p>
</li>
<li><p><a href="https://cran.r-project.org/web/packages/MiSPU/index.html" target=&ldquo;blank&rdquo;>MiSPU</a>:  R package that presents a novel global testing method called aMiSPU, that is highly adaptive and thus high powered across various scenarios, alleviating the issue with the choice of a phylogenetic distance. The corresponding paper is <a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-016-0302-3" target=&ldquo;blank&rdquo;>Wu, Chen, Kim and Pan, 2016</a>.</p>
</li>
<li><p><a href="https://github.com/ChongWu-Biostat/GLMaSPU" target=&ldquo;blank&rdquo;>GLMaSPU</a>: R package that makes it incredibly easy to implement some testing methods under high-dimensional generalized linear models. The corresponding paper is our 2019 Stat Sinica paper. </p>
</li>
<li><p><a href="glmtlp.html" target=&ldquo;blank&rdquo;>glmtlp</a>: R package that makes it easy to implement the truncated lasso penalty under a generalized linear model framework. This package is similar to <a href="https://web.stanford.edu/~hastie/glmnet/glmnet_alpha.html" target=&ldquo;blank&rdquo;>glmnet</a> but can be applied with a non-convex penalty. </p>
</li>
</ul>
<div class="infoblock">
<div class="blockcontent">
<p>To help researchers from other field use our newly developed method, we have created and maintained the following software and pipeline. We assume no prior knowledge in R and all the following software can be run easily and smoothly once the required packages are installed. Please send me an email ï¼ˆcwu3@fsu.edu) if you have find any bugs when using them.</p>
</div></div>
<ul>
<li><p><a href="IWAS.html" target=&ldquo;blank&rdquo;>IWAS</a>: A software for implementing Imaging-Wide Association Studies (IWAS). The corresponding paper is <a href="https://www.ncbi.nlm.nih.gov/pubmed/28736311" target=&ldquo;blank&rdquo;>our 2017 NeuroImage paper</a>.</p>
</li>
<li><p><a href="TWAS.html" target=&ldquo;blank&rdquo;>TWAS-aSPU</a>: A more powerful gene-based association test to integrate single set or multiple sets of eQTL data with GWAS individual-level data or summary statistics. The corresponding paper is <a href="http://www.genetics.org/content/genetics/early/2017/09/11/genetics.117.300270.full.pdf" target=&ldquo;blank&rdquo;>our 2017 Genetics paper</a>.</p>
</li>
<li><p><a href="aspupath2.html" target=&ldquo;blank&rdquo;>aSPUpath2</a>: A new pathway-based method for integrating eQTL data with GWAS summary statistics. This can be viewed as an extension of TWAS to the pathway-based analysis. The corresponding paper is <a href="https://www.ncbi.nlm.nih.gov/pubmed/29411426" target=&ldquo;blank&rdquo;>our 2018 Genetic Epidemiology paper</a>.</p>
</li>
<li><p><a href="egmethyl.html" target=&ldquo;blank&rdquo;>egmethyl</a>: A new gene-based test for integrating enhancer-promoter interactions and DNA methylation data with GWAS summary data. The corresponding paper is <a href="https://doi.org/10.1093/bioinformatics/btz161" target=&ldquo;blank&rdquo;>our 2019 Bioinformatics paper</a>.</p>
</li>
</ul>
<div class="infoblock">
<div class="blockcontent">
<p>We create a <a href="https://github.com/ChongWuLab" target=&ldquo;blank&rdquo;>GitHub lab page</a> for the software written by the group members</p>
</div></div>
<ul>
<li><p><a href="https://github.com/ChongWuLab/CMO" target=&ldquo;blank&rdquo;>CMO</a>: Cross Methylome Omnibus (CMO) integrates genetically regulated DNAm in enhancers, promoters, and the gene body to identify additional disease-associated genes.</p>
</li>
<li><p><a href="https://github.com/ChongWuLab/FOGS" target=&ldquo;blank&rdquo;>FOGS</a>: FOGS is  a powerful fine-mapping method that prioritizes putative causal genes by accounting for local LD in TWAS results</p>
</li>
</ul>
</td>
</tr>
</table>
</body>
</html>
