<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title></title>
</head>
<body>
<div id="layout-content">
<h1>TWAS-aSPU</h1>
<h2>Integrating eQTL and GWAS data</h2>
<p>Two new gene-based association analysis methods, called PrediXcan and TWAS for GWAS individual-level and summary data respectively, were recently proposed to integrate GWAS with eQTL data, alleviating two common problems in GWAS by boosting statistical power and facilitating biological interpretation of GWAS discoveries. Based on a novel reformulation of PrediXcan and TWAS, <b>we propose a more powerful gene-based association test to integrate single set or multiple sets of eQTL data with GWAS individual-level data or summary statistics</b>. As demonstrated in our simulations and real data analyses and hopefully will be demonstrated in your own studies, the proposed method uncovered more known or novel trait-associated genes, showcasing much-improved performance of our proposed method. Please cite the following manuscript for using the TWAS-aSPU method:</p>
<div class="infoblock">
<div class="blockcontent">
<p>Xu, Z., Wu, C., Wei, P., and Pan, W. (2017+). <a href="http://www.genetics.org/content/genetics/early/2017/09/11/genetics.117.300270.full.pdf" target=&ldquo;blank&rdquo;>A powerful framework for integrating eQTL and GWAS summary data</a>. Accepted by <i>Genetics</i>, early online.</p>
</div></div>
<p>For questions or comments regarding methods, contact Wei Pan (<a href="mailto:weip@biostat.umn.edu" target=&ldquo;blank&rdquo;>weip@biostat.umn.edu</a>);
For questions or comments regarding data & codes, contact Chong Wu (<a href="mailto:chongwu@umn.edu" target=&ldquo;blank&rdquo;>chongwu@umn.edu</a>).</p>
<h2>Installation</h2>
<p>Note: Since the proposed method can be treated as the extension of <a href="http://gusevlab.org/projects/fusion/" target=&ldquo;blank&rdquo;>TWAS</a>, some steps are exactly the same and taken from the <a href="http://gusevlab.org/projects/fusion/" target=&ldquo;blank&rdquo;>TWAS website</a>.</p>
<ul>
<li><p>Download and unpack the (1000 Genomes)  <a href="https://data.broadinstitute.org/alkesgroup/FUSION/LDREF.tar.bz2" target=&ldquo;blank&rdquo;>LD reference data</a>:</p>
</li>
</ul>
<div class="codeblock">
<div class="blockcontent"><pre>
wget https://data.broadinstitute.org/alkesgroup/FUSION/LDREF.tar.bz2
tar xjvf LDREF.tar.bz2
</pre></div></div>
<ul>
<li><p>Download the weights from the <a href="http://gusevlab.org/projects/fusion/" target=&ldquo;blank&rdquo;>TWAS website</a> or <a href="https://github.com/hakyimlab/PrediXcan" target=&ldquo;blank&rdquo;>PrediXcan</a> website.</p>
</li>
</ul>
<ul>
<li><p>Download and unpack the <a href="https://github.com/ChongWu-Biostat/TWAS" target=&ldquo;blank&rdquo;>Scripts and examples</a> (Using clone or download option at GitHub).</p>
</li>
</ul>
<div class="codeblock">
<div class="blockcontent"><pre>
git clone https://github.com/ChongWu-Biostat/TWAS.git
</pre></div></div>
<ul>
<li><p>Launch R and install required libraries:</p>
</li>
</ul>
<div class="codeblock">
<div class="blockcontent"><pre>
install.packages('optparse','RColorBrewer','data.table','matlib','Rcpp','RcppArmadillo','bigmemory','mvtnorm','MASS')
install.packages('plink2R-master/plink2R/',repos=NULL)
if (!require("devtools"))
install.packages("devtools")
devtools::install_github("ChongWu-Biostat/aSPU2")
devtools::install_github("gabraham/plink2R/plink2R")
</pre></div></div>
<h2>Typical analysis and output</h2>
<p>The TWAS-aSPU analysis takes gene expression based external weights and disease GWAS summary statistics to identify significant genes. We will use the <a href="http://web.pasteur-lille.fr/en/recherche/u744/igap/igap_download.php" target=&ldquo;blank&rdquo;>IGAP Alzheimer's summary data</a> (Lambert et al. 2013) as an example to illustrate how to use our methods. This example assumes you have setup the required environment and data as illustrated in the previous section. All analyses are based on R/3.3.1.</p>
<h3>Input: GWAS summary statistics</h3>
<p>At a minimum, we need a summary rds file with a header row containing the following fields:</p>
<ol>
<li><p>SNP_map – SNP identifier (CHR:BP)</p>
</li>
<li><p>A1 – first allele (effect allele, should be capitalized)</p>
</li>
<li><p>A2 – second allele (other allele, should be capitalized)</p>
</li>
<li><p>Z – Z-scores, sign with respect to A1.</p>
</li>
</ol>
<p>Additional columns are allowed but will be ignored. We recommend removing the additional columns before analysis to save space.</p>
<p><b>Note:</b> The performance of TWAS-aSPU depends on the density of summary-level data. We highly recommend running TWAS-aSPU with raw summary-level data. Pre-process step such as pruning and restricting to top SNPs may harm the performance.</p>
<h3>Input: external weights</h3>
<p>The pre-computed external weights can be downloaded from <a href="http://gusevlab.org/projects/fusion/" target=&ldquo;blank&rdquo;>TWAS</a> (Gusev et al. 2016) or <a href="https://github.com/hakyimlab/PrediXcan" target=&ldquo;blank&rdquo;>PrediXcan</a> websites.</p>
<h3>Performing the TWAS-aSPU</h3>
<p>After we prepared the data, we can run IWAS via the following single line.</p>
<div class="codeblock">
<div class="blockcontent"><pre>
Rscript TWAS_aSPU.R \
--sumstats ./Example/IGAP_chr22.rds \
--out ./Example/example_res.rds \
--weights ./WEIGHTS/NTR.BLOOD.RNAARR.pos \
--weights_dir ./WEIGHTS/ \
--ref_ld ./LDREF/1000G.EUR. \
--gene_list ./Example/gene_list.txt \
--chr 22
</pre></div></div>
<p>This should take around one or two minutes, and you will see some intermediate steps on the screen. If everything works, you will see a file <i>example_res.rds</i> under the <i>.Example</i> and <i>output.txt</i> in the working directory.</p>
<p>Through TWAS_aSPU.R, we perform the following steps: (1) combine the GWAS and reference SNPs and remove ambiguous SNPs. (2) impute GWAS Z-scores for any reference SNPs with missing GWAS Z-scores via the IMPG algorithm ([https:<i></i>academic.oup.com<i>bioinformatics</i>article<i>30</i>20<i>2906</i>2422225/Fast-and-accurate-imputation-of-summary-statistics Pasaniuc et al. 2014); (3) perform TWAS-aSPU; (4) report results and store them in the working directory.</p>
<h3>Output: Gene-disease association</h3>
<p>The results are stored in a user-defined output file. For illustration, we explain the meaning of each entry in the first two lines of the output.</p>
<table id="VariExp">
<tr class="r1"><td class="c1">Col. num. </td><td class="c2"> Column name   </td><td class="c3"> Value  </td><td class="c4"> Explanations </td></tr>
<tr class="r2"><td class="c1">1 </td><td class="c2"> gene         </td><td class="c3"> A4GALT </td><td class="c4"> Feature/gene identifier, taken from gene_list file </td></tr>
<tr class="r3"><td class="c1">2 </td><td class="c2"> CHR        </td><td class="c3"> 22     </td><td class="c4"> Chromosome </td></tr>
<tr class="r4"><td class="c1">3 </td><td class="c2"> P0            </td><td class="c3"> 41088126 </td><td class="c4"> Gene start (from hg19 list from plink website) </td></tr>
<tr class="r5"><td class="c1">4 </td><td class="c2"> P1        </td><td class="c3">   43116876  </td><td class="c4"> Gene end (from hg19 list from plink website) </td></tr>
<tr class="r6"><td class="c1">5 </td><td class="c2"> #nonzero_SNPs </td><td class="c3"> 5 </td><td class="c4"> Number of non-zero weight SNPs </td></tr>
<tr class="r7"><td class="c1">6 </td><td class="c2"> TWAS_asy </td><td class="c3"> 0.69 </td><td class="c4"> TWAS p-value with imaging based external weight. The p-value is based on asymptotic distribution. </td></tr>
<tr class="r8"><td class="c1">7 </td><td class="c2"> SSU_asy </td><td class="c3"> 0.61 </td><td class="c4"> SSU p-value with imaging based external weight. The p-value is based on asymptotic distribution. </td></tr>
<tr class="r9"><td class="c1">8-23</td><td class="c2"> SPU </td><td class="c3"> 0.69</td><td class="c4"> SPU or aSPU p-values. The results are based on simulations.
</td></tr></table>
<p><b>Note:</b> We only store the results for genes with external weights. The genes without external weights will be ignored in the output file.</p>
<h2>Further Analyses</h2>
<h3>Testing for effect in multiple external weights</h3>
<p>There may be compelling reasons to take advantage of multiple sets of weights based on multiple correlated endophenotypes. First, the statistical advantages of joint analysis of multiple traits include possibly increasing statistical power and more precise parameter estimates, alleviating the burden of multiple testing. Biologically, joint analysis of multiple traits addresses the issue of pleiotropy (i.e. one locus influencing multiple traits), providing biological insight into molecular mechanisms underlying the disease or trait. Second, the above conclusions are expected to carry over to the current context of analysis of multiple endophenotypes. In our current version of the software, we provide an adaptive test, called aSPUO, which can combine the information from multiple sets of weights simultaneously. This test also covers TWAS-omnibus as a special situation. You can call <i>aSPUO</i> via aSPU2 package. Note that this procedure is very time-consuming.</p>
<h3>Testing with the individual level GWAS data</h3>
<p>TWAS-aSPU can be applied to individual level GWAS data as well. The main idea is the same. Specifically, we can calculate the score and its corresponding covariance matrix for SNPs in the gene regions. Then we can apply <i>aSPU2</i> package with pre-computed imaging endophenotype based external weights provided on this website. Since there some restrictions sharing individual level GWAS data, we do not provide any example regarding applying TWAS-aSPUO to individual level data. However, this should be a relatively easy task.</p>
<h2>Command-line parameters</h2>
<h3>TWAS_aSPU.R</h3>
<table id="Command">
<tr class="r1"><td class="c1">Flag   </td><td class="c2"> Usage  </td><td class="c3"> Default </td></tr>
<tr class="r2"><td class="c1">--sumstats </td><td class="c2"> ummary statistics (rds file and must have SNP and Z column headers) </td><td class="c3"> Required </td></tr>
<tr class="r3"><td class="c1">--out </td><td class="c2"> Path to output file </td><td class="c3"> Required </td></tr>
<tr class="r4"><td class="c1">--weights </td><td class="c2"> File listing molecular weight (rds files and must have columns ID,CHR,P0,P1, Weights) </td><td class="c3"> Required </td></tr>
<tr class="r5"><td class="c1">--ref_ld </td><td class="c2"> Reference LD files in binary PLINK format </td><td class="c3"> Required </td></tr>
<tr class="r6"><td class="c1">--gene_list </td><td class="c2"> Gene sets we want to analyze, currently only gene sets from a single chromosome are supported </td><td class="c3"> Required </td></tr>
<tr class="r7"><td class="c1">--max_nperm </td><td class="c2"> maximum number of permutation for aSPU or daSPU </td><td class="c3"> 1000000
</td></tr></table>
<p><b>note:</b> A single layer/loop of Monte Carlo simulation is used to obtain the \(p\)-values of all the SPU, aSPU, and daSPU tests simultaneously. we use an adaptive way to select the number of simulations and calculate \(p\)-values efficiently. --max_nperm is the upper bound for the number of simulations.</p>
<h2>FAQ</h2>
<p><b>TWAS-aSPU looks similar to TWAS and PrediXcan. What's the difference between them?</b></p>
<p>By noting that TWAS and PrediXcan are the same as a weighted Sum test with gene expression based weights, we propose to use aSPU, a more powerful and adaptive test, to conduct association testing. Since aSPU covers the (weighted) Sum test as a special case, we can get the results for TWAS or PrediXcan after running our method as well. We demonstrate that our new method is more powerful and identify some new genes in some real data analyses. We expect this to be generally true and hope you can let us know your results if you apply both TWAS and IWAS on your real data analyses.</p>
<p><b>What related software is available?</b></p>
<p>Our methods are related to the following method by other two groups.
Two methods are highly correlated with two well-known groups.  <a href="https://github.com/hakyimlab/MetaXcan" target=&ldquo;blank&rdquo;>MetaXcan</a> and  <a href="https://github.com/hakyimlab/PrediXcan" target=&ldquo;blank&rdquo;>PrediXcan</a>  (Gamazon et al 2015) developed by the Im lab perform gene-based association tests with and without summary data. <a href="http://gusevlab.org/projects/fusion/" target=&ldquo;blank&rdquo;>TWAS</a> by Gusev performs gene-based association tests with individual-level or summary statistics.  MetaXcan and TWAS are similar. The main difference between TWAS and MetaXcan is that they use a slightly different way and different reference data to construct weights. We recommend you to check their websites to download other gene-expression based weights as well.</p>
<p>Instead of using gene-expression based external weight, we may construct weights based on other endophenotypes. See our <a href="www.wuchong.org/IWAS.html" target=&ldquo;blank&rdquo;>IWAS</a> for more details.</p>
<p><b> What QC is performed internally when using TWAS-aSPU?</b></p>
<p>TWAS-aSPU performs a similar quality control steps as TWAS did. We automatically match up SNPs, remove <a href="https://www.snpedia.com/index.php/Ambiguous_flip" target=&ldquo;blank&rdquo;>ambiguous markers</a> (A/C or G/T) and flip alleles to match the reference data.</p>
<h2>Acknowledgements</h2>
<p>This research was supported by NIH grants R21AG057038, R01HL116720, R01GM113250 and R01HL105397, and by the Minnesota Supercomputing Institute; ZX was supported by a University of Minnesota MnDRIVE Fellowship and CW by a University of Minnesota Dissertation Fellowship.</p>
<h2>References</h2>
<ul>
<li><p>Lambert, JC., Ibrahim-Verbaas, CA., Harold, D., Naj, AC., Sims, R., Bellenguez, C., DeStafano, AL., Bis, JC., Beecham, GW., Grenier-Boley, B., et al. (2013) Meta-analysis of 74,046 individuals identifies 11 new susceptibility loci for Alzheimer’s disease. <i>Nat. Genet.</i> 45, 1452–1460.</p>
</li>
<li><p>Pasaniuc, B., Zaitlen, N., Shi, H., Bhatia, G., Gusev, A., Pickrell, J., Hirschhorn, J., Strachan, D.P., Patterson, N. and Price, A.L. (2014). Fast and accurate imputation of summary statistics enhances evidence of functional enrichment. <i>Bioinformatics</i> 30, 2906- 2914.</p>
</li>
<li><p>Gusev, A. et al. (2016) Integrative approaches for large-scale transcriptome-wide association studies. <i>Nat Genet.</i> 48, 245-252.</p>
</li>
<li><p>Gamazon, E.R. et al. (2015) A gene-based association method for mapping traits using reference transcriptome data. <i>Nat. Genet.</i> 47, 1091-1098.</p>
</li>
<li><p>Alvaro Barbeira, Kaanan P Shah, Jason M Torres, Heather E Wheeler, Eric S Torstenson, Todd Edwards, Tzintzuni Garcia, Graeme I Bell, Dan Nicolae, Nancy J Cox, Hae Kyung Im. (2016) <a href="http://biorxiv.org/content/early/2017/05/21/045260" target=&ldquo;blank&rdquo;>MetaXcan: Summary Statistics Based Gene-Level Association Method Infers Accurate PrediXcan Results</a>.</p>
</li>
</ul>
<h2>License</h2>
<p>Maintainer: <a href="index.html" target=&ldquo;blank&rdquo;>Chong Wu</a> (wuxx0845@umn.edu)</p>
<p><a href="http://opensource.org/licenses/MIT" target=&ldquo;blank&rdquo;>MIT</a></p>
<p>Copyright (c) 2013-present, Chong Wu (wuxx0845@umn.edu), Zhiyuan Xu (xuxx0284@umn.edu), Wei Peng (PWei2@mdanderson.org) &amp; Wei Pan(weip@biostat.umn.edu).</p>
<div id="footer">
<div id="footer-text">
Page generated 2018-02-24 23:20:41 CST, by <a href="https://github.com/wsshin/jemdoc_mathjax" target="blank">jemdoc+MathJax</a>.
</div>
</div>
</div>
</body>
</html>
